lib/task-data/tasks/install-suse.js

```diff
@@ -11,12 +11,35 @@ module.exports = {
         osType: 'linux', //readonly options, should avoid change it
         profile: 'install-suse.ipxe',
         installScript: 'suse-autoinst.xml',
-        installScriptUri: '{{ api.templates }}/{{ options.installScript }}?nodeId={{ task.nodeId }}',
+        installScriptUri: '{{ api.templates }}/{{ options.installScript }}?nodeId={{ task.nodeId }}', //jshint ignore: line
         hostname: 'localhost',
         comport: 'ttyS0',
         repo: '{{file.server}}/distribution/{{options.version}}/repo/oss/',
         rootPassword: "RackHDRocks!",
-        kargs:{}
+        kargs:{},
+
+        progressMilestones: {
+            //jshint ignore: start
+            requestProfile:     { value: 1, description: 'Enter ipxe and request OS installation profile' },
+            enterProfile:       { value: 2, description: 'Enter profile, start to download installer'},
+            startInstaller:     { value: 3, description: 'Start installer and prepare installation' },
+            preConfig:          { value: 4, description: 'Enter Pre OS configuration'},
+            postPartitioning:   { value: 5, description: 'Finished partitioning and mounting, start package installation'},
+            chroot:             { value: 6, description: 'Finished package installation, start first boot'},
+            // Now install SUSE task is designed to exist earlier before the postConfig milestone,
+            // this is because the postConfig is triggered during OS first boot stage, but the first
+            // boot will re-download the ipxe script and start a refresh installation rather than
+            // continue the previous installation. This is the reason why the postConfig milestone
+            // is removed.
+            // This is a defect for RackHD, multiple reboot shouldn't occur during the life of the
+            // same task.
+            // TODO: Try to find a way to let RackHD supports multiple reboot during a task
+            // execution.
+            //
+            // postConfig:      { value: 7, description: 'Finished system configuration, and booted into system'},
```

Comment:
Actually I intentionally leave this comment here, I have wrote a lot words  to explain above this line and also have added a TODO. This is a improve point for RackHD, if it is implemented, we can then uncomment this line.